src/renderers/common/Renderer.js

```diff
@@ -2225,6 +2225,21 @@ class Renderer {
 
 	}
 
+	/**
+	 * Resets the renderer to the initial state before WebXR started.
+	 *
+	 */
+	XRResetState() {
+
+		this.backend.setXRTarget( null );
+		this.setOutputRenderTarget( null );
+		this.setRenderTarget( null );
+
+		this._frameBufferTarget.dispose();
+		this._frameBufferTarget = null;
+
+	}
```

Comment:
Not a fan of adding more and more public functions to `Renderer` that we'll have to maintain...

But, if there's no other way, how about calling this `resetXRState()` instead?

Reply:
One issue is that XR rendering requires specific rendering setups that can't be configured with the usual renderer API. It becomes more complicated the more XR features (like layers, multi-view) the engine supports.

The specific interface `XRManager` requires does not have to be public. We could name the method `_resetXRState()` and treat it as a pure internal method.

Reply:
`_resetXRState()` sounds good to me 👍